src/Ledger/Epoch.lagda

```diff
@@ -74,45 +76,43 @@ its results, i.e:
 \begin{code}
   EPOCH : let
       open EpochState eps hiding (es)
-      open RatifyState fut using (removed) renaming (es to esW)
       -- ^ this rolls over the future enact state into es
       open LState ls; open UTxOState utxoSt; open CertState certState
-      open PState pState; open DState dState; open GState gState
-      open Acnt acnt
 
```

Comment:
Removed a couple of instances of `open` for records here, at the expense of using accessors below.